docs/setup-guide/scale-out-local-dev-setup.md

```diff
@@ -0,0 +1,102 @@
+# Setting up local dev environment for scale out
+
+## Scenarios
+
+1. Two Tenant Partitions
+
+1. Single Resource Partition (2/19/2021)
+
+1. HA proxy
+
+## Prerequsite
+
+1. 3 dev box (tested on ubuntu 16.04), 1 for RP, 2 for TPs. Record ip as TP1_IP, TP2_IP, RP_IP
```

Comment:
note the TP and proxy can be put into one box, RPs needs to be separated into different boxes

Reply:
Right now, this manual is for internal reference only. Will need more test and rephrase if we need it in master. Also only for 2TP/1RP right now.